File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/orazioc17.py

def run():
    print('Bienvenidos a mi FizzBuzz\n')
    for number in range(1, 101):
        string = ''
        string += 'fizz' if (number % 3 == 0) else ''
        string += 'buzz' if (number % 5 == 0) else ''
        print(string) if string != '' else print(number)


if __name__ == '__main__':
    run()

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/Orazioc17.java
public class Orazioc17 {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++) {
            String palabra = "";

            palabra = (i % 3 == 0)?palabra+"fizz":palabra;
            palabra = (i % 5 == 0)?palabra+"buzz":palabra;

            palabra = (palabra.equals(""))?palabra+i:palabra;

            System.out.println(palabra);
        }
    }
}